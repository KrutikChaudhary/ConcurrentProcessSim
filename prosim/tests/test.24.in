2 3
Proc1 3 -1
BLOCK 1
DOOP 2
HALT

Proc2 2 -1
DOOP 5
HALT
