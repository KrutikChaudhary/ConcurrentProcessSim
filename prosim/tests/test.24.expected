00000: process 1 new
00000: process 1 blocked
00000: process 2 new
00000: process 2 ready
00000: process 2 running
00001: process 1 ready
00001: process 2 ready
00001: process 1 running
00003: process 1 finished
00003: process 2 running
00006: process 2 ready
00006: process 2 running
00007: process 2 finished
Process 1: Run time 2, Block time 1, Wait time 0
Process 2: Run time 5, Block time 0, Wait time 2
